packages/tinacms/src/components/TinaProvider.tsx

```diff
@@ -60,6 +61,7 @@ export const TinaProvider: React.FC<TinaProviderProps> = ({
         <Alerts alerts={cms.alerts} />
         {enabled && styled && <Theme />}
         {enabled && cms.toolbar && <Toolbar />}
+        <MediaManager />
```

Comment:
should this be `enabled &&`?

Reply:
It could be. It's not rendered until `cms.media.open` is called so it's fine, but we might as well